static-assets/scripts/main.js

```diff
@@ -1099,6 +1099,19 @@
     }
   ]);
 
+  app.controller('AboutCtrl', [
```

Comment:
Make sure to clean up any requests on this side that will now be done on the react side

Reply:
That requests will be cleaned on the next reactify(account) because they were using the same controller